src/js/i18n/en-US.yml

```diff
@@ -155,6 +156,15 @@ careOptsFrontend:
             updated_past_three_days {Actions updated in the last 3 days by any clinician in your Group(s). States included are To Do and In Progress.}
             done_last_thirty_days {Actions set to Done by any clinician in your Group(s) in the last 30 days.}
           }
+  programs:
+    list:
+      programsAllViews:
+        emptyView: No Programs
+        itemView:
+          'off': "Off"
```

Comment:
what's with the quotes on the left side?  If they're needed let's change the text to `offText` or something so they're not

Reply:
syntax highlighter made it look like they were reserved words, so I just did this to make it happy

Reply:
![image](https://user-images.githubusercontent.com/2028470/66860731-b2933400-efc8-11e9-90fc-c39eb6d84835.png)